shared/storage/certpool/bundler.go

```diff
@@ -0,0 +1,101 @@
+package certpool
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"log"
+
+	"github.com/darvaza-proxy/darvaza/shared/x509utils"
+)
+
+var (
+	_ x509utils.Bundler = (*CertPool)(nil)
+)
+
+// Bundler uses two CertPoolers to bundler keys and certificates
+type Bundler struct {
+	Roots x509utils.CertPooler
+	Inter x509utils.CertPooler
+}
+
+// Bundle bundles a key and a certificate into a *tls.Certificate
+func (b *Bundler) Bundle(cert *x509.Certificate, key x509utils.PrivateKey) (
+	*tls.Certificate, error) {
```

Comment:
I know it is valid, but please do not break lines like this. It is very annoying to me.